tcms/testruns/templates/testruns/get.html

```diff
@@ -79,25 +72,35 @@ <h2 class="card-pf-title" style="text-align: left">
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa-calendar"></span>{% trans 'Started at' %}:
                     {{ object.start_date }}
+                    <span class="start-date">
+                            {{ object.start_date|default:'-' }}
+                        </span>
+                    {% if not object.start_date %}
+                        <button class="btn btn-success btn-xs" type="button"        id="start-button"
+                                data-toggle="tooltip"          data-placement="top" title="{% trans 'Start' %}"
+                        style="float:right;">
+                            <i class="fa fa-check" style="color: white; margin: auto; "></i>
+                        </button>
+                    {% endif %}
                 </h2>
 
                 <h2 class="card-pf-title" style="text-align: left">
                     <span class="fa fa fa-calendar-o"></span>{% trans 'Planned stop' %}:
                     {{ object.planned_stop|default:'-' }}
                 </h2>
 
-                <h2 class="card-pf-title" style="text-align: left">
+                <h2 class="card-pf-title" style="text-align: left; overflow: hidden;">
                     <span class="fa fa-calendar-check-o"></span>{% trans 'Finished at' %}:
                     <span class="stop-date">
-                        {% if object.stop_date %}
-                            {{ object.stop_date }}
-                        {% else %}
-                            -
-                        {% endif %}
+                        {{ object.stop_date|default:'-' }}
                     </span>
+                    {% if not object.stop_date %}
+                        <button class="btn btn-warning btn-xs pull-right" type="button"        id="stop-button"
+                                data-toggle="tooltip"                     data-placement="top" title="{% trans 'Stop' %}" style="margin-bottom: 1rem;">
+                            <i class="fa fa-check" style="color: white; margin: auto; "></i>
+                        </button>
+                    {% endif %}
                 </h2>
-
-                <div class="card-pf-body"></div>
```

Comment:
This is a mandatory element, eventhough it is empty. Bring it back.

Reply:
@atodorov done